README.md

```diff
@@ -60,11 +60,12 @@ This means we follow the same installation than above, but using the
 ```sh
 λ opam switch 4.02.3+buckle-master
 λ eval `opam config env`
-λ opam install odoc
-λ odoc --version
-1.2.0
+λ opam pin add odoc.dev git+https://github.com/ocaml/odoc
+
 ```
 
+> BuckleScript support currently requires the latest development version of odoc.
```

Comment:
This feels like something we should include in the very first paragraph of this title. I'd want to know from the get go that I need the dev version of odoc.

Good catch otherwise!

Reply:
👍